### ISSUE TITLE
[html-indent] Add possibility for half indentation on attributes

### ISSUE THREAD

**Issue by aberonni** [vuejs/eslint-plugin-vue#784]
**Please describe what the rule should do:**
This isn't quite a new rule proposal but it isn't a bug either.

I'd like to have the possibility to have 4 space indentation on all html, except for attributes which should be indented by 2 spaces. I'm not sure whether this is feasible or whether it is something that the community is interested in, but I guess it is worth a try.

**What category should the rule belong to?**

- [X] Enforces code style
- [ ] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
```vue
<template>
    <!-- ✓ GOOD -->
    <div class="foo">
        Hello.
    </div>
    <div class="foo">
        Hello.
    </div>
    <div
      class="foo"
      :foo="bar"
    >
        World.
    </div>
    <div
      id="a"
      class="b"
      :other-attr="{
        aaa: 1,
        bbb: 2
      }"
    />

    <!-- ✗ BAD -->
    <div
        id="a"
        class="b"
    />
</template>
```

**Comment by ota-meshi** [comment]
Thank you for your suggestions. Sorry for the late reply.

I don't know how many people like this style, but I don't implement this because I think it would be complicated to configure.

I close this issue.
